libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -473,7 +470,23 @@ void elasticsearch_plugin::plugin_initialize(const boost::program_options::varia
    }
    if (options.count("elasticsearch-start-es-after-block")) {
       my->_elasticsearch_start_es_after_block = options["elasticsearch-start-es-after-block"].as<uint32_t>();
-   }   
+   }
+   if (options.count("elasticsearch-operation-string")) {
+      my->_elasticsearch_operation_string = options["elasticsearch-operation-string"].as<bool>();
+   }
+   if (options.count("elasticsearch-mode")) {
+      my->_elasticsearch_mode = options["elasticsearch-mode"].as<std::string>();
+   }
+
+   if(my->_elasticsearch_mode != "only_query") {
+      if (my->_elasticsearch_mode == "all")
+         my->_elasticsearch_operation_string = true;
```

Comment:
Silently overriding this will cause confusion.
IMO add log output if user has configured this to false. Or perhaps fail to make this very explicit.

Reply:
Agree, i am now not allowing the mode to be all if operation_string is false so no silent changes are made.

https://github.com/bitshares/bitshares-core/pull/1725/commits/5838a38fbc93b2673e7c96640abe15a5caa4533e